deploy-board/deploy_board/templates/groups/health_check_activities.tmpl

```diff
@@ -42,6 +42,10 @@
                     <span class="deployToolTip pointer-cursor {{ health_check.status | healthCheckStatusIcon }}"></span>
                     {% if health_check.status == "SUCCEEDED" %}
                         STATE SUCCEEDED
+                    {% elif health_check.status == "QUALIFIED" %}
+                        QUALIFIED (PENDING_TERMINATION)
+                    {% elif health_check.status == "TELETRAAN_STOP_REQUESTED" %}
+                        QUALIFIED (TERMINATED)
```

Comment:
Technically it's not necessary terminated.

Reply:
^ @liyaqin1 
Agreed, maybe just `QUALIFIED`?

Reply:
I don't see the difference between terminated and TELETRAAN_STOP_REQUESTED. If we want a strict terminated, we need to add a status in rodimus side. From my point of view, TELETRAAN_STOP_REQUESTED is the last status in the whole process. If you want me to update in rodimus side, feel free to let me know.